languages/elixir/exercises/concept/maps/.docs/after.md

```diff
@@ -0,0 +1,73 @@
+Great work, knowing how to effectivly use [maps][maps] is an import skill to use Elixir fluently.
+
+## Key points
+
+- [Maps][maps] are a data structure that holds key-value pairs.
+- Keys can be of any type, but must be unique.
+- Values can be of any type, they do not have to be unique.
+- [Maps][maps] do not guarantee the order of their contents despite appearing so.
+  - Their underlying implementation gives this misperception:
+    - At small sized (<=32 entries), they are implemented as an ordered [Keyword list][keyword-list]
+    - At larger sizes (>32 entries), they are implemented as a [hash array mapped trie][hamt] &#91;[1][stackoverflow]&#93;
+- [Maps][maps] can be declared with a literal form:
```

Comment:
This says here, that maps _can_ be declared with a literal form. Is there also another way to declare maps? If so, it is perhaps useful to show how to use that variant and discuss which one is preferred (and why).

Reply:
Ah, for sure.